setup.py

```diff
@@ -15,19 +15,25 @@
 #
 import io
 import os
+import pathlib
+
 
 import setuptools  # type: ignore
 
-package_root = os.path.abspath(os.path.dirname(__file__))
+package_root = pathlib.Path(__file__).parent.resolve()
 
 name = "google-generativeai"
 
 description = "Google Generative AI High level API client library and tools."
 
-version = {}
-with open(os.path.join(package_root, "google/generativeai/version.py")) as fp:
-    exec(fp.read(), version)
-version = version["__version__"]
+def get_version():
```

Comment:
Would this be necessary if we could import `version.py` without any other deps getting in the way?